### DISCUSSION TITLE
Add VERCEL_PROJECT_NAME to System Environment Variables

### DISCUSSION THREAD

**Discussion by yannispgs** [vercel/community#2868] [Idea]
### Idea

Hi,

I have 1 Github repository used by multiple Vercel Projects as I stored multiple Github Bots inside. However, I do not want to have 1 new deployment in each Project for every new commit pushed. I have a script that check commit messages and builds the list of "bots" projects to update based on modified files.

However, I cannot use that script in Vercel because there is no way to know, from an Ignored Build Step script, in what project it is currently running. Indeed, it is not available in the System Environment Variables.

Having this value as a new variable would be highly appreciated.

**Comment by amyegan** [comment]
Nice idea, @yannispgs! As a workaround you could use `VERCEL_URL` or `VERCEL_BRANCH_URL` to figure out what the current project is. I know that isn't as convenient as a dedicated env var, but I hope it can get you unstuck for the time being.

**Reply by yannispgs** [reply]
That could work indeed, thanks ! But I still hope I'll see this new System Environment Variable in the changelog one day 😁
